src/common/prntbase.cpp

```diff
@@ -626,15 +626,30 @@ bool wxPrintout::HasPage(int page)
 
 void wxPrintout::GetPageInfo(int *minPage, int *maxPage, int *fromPage, int *toPage)
 {
-    *minPage = 1;
-    *maxPage = DEFAULT_MAX_PAGES;
-    *fromPage = 1;
-    *toPage = 1;
+    if(minPage)
```

Comment:
I don't think we need those, the parameters are guaranteed to be non-null here AFAICS.